src/main/java/offeneBibel/parser/OffeneBibelParser.java

```diff
@@ -281,6 +284,9 @@ public Rule LineQuote() {
                         BibleText(),
                         Verse(),
                         Note(),
+                        Sequence("“", createOrAppendTextNode("“")),
+                        Sequence("‘", createOrAppendTextNode("‘")),
```

Comment:
Nachdem ich nicht genau weiß wie LineQuotes funktionieren sollen, weiß ich nicht, wie ich LineQuotes korrigieren soll, wo innerhalb eines LineQuotes ein Zitat (erster oder zweiter Ebene) auf und in einem späteren wieder zu geht.

Das war mehr oder weniger ein quick&dirty Fix, um das minimalinvasiv geparst zu bekommen (ohne die ganzen anderen Stellen an denen ich im Wikitext Anführungszeichen korrigiert habe auch mit "durchzuwinken"). Kann man sicher schöner lösen (ggf. auch mit einem Parser mit mehreren Läufen, oder LineQuotes/SecondaryContent anders parsen). Oder natürlich definieren dass sowas nicht sein darf, dann müsste man aber wissen was anstelle besser wäre.